raiden/raiden_service.py

```diff
@@ -150,12 +151,20 @@ class RaidenService(object):
     def __init__(self, chain, private_key_bin, transport, discovery, config):
         if not isinstance(private_key_bin, bytes) or len(private_key_bin) != 32:
             raise ValueError('invalid private_key')
-
```

Comment:
Why remove the whitespace here? It's helpful for reading the code to have whitespace between if statements.